src/main-oe/sgxlkl_run_oe.c

```diff
@@ -88,6 +88,16 @@ typedef struct ethread_args
     oe_enclave_t* oe_enclave;
 } ethread_args_t;
 
+// CV to wait for the terminating ethreads, which shuts down the enclave
+static pthread_cond_t terminating_ethread_exited_cv;
+static pthread_mutex_t terminating_ethread_exited_mtx;
+
+// Exit status returned from the enclave
+static long enclave_return_status = 0;
```

Comment:
Initialise to `LONG_MAX` so we can differentiate a real return value?

Reply:
Done.